src/mocks/freedom-mocks.ts

```diff
@@ -20,6 +20,11 @@ class MockCore {
 
   public getId = () => { return ['useless']; }
 
+  public getLogger = (tag) => {
```

Comment:
Can you comment on what this is and why its needed?  We don't have any getLogger method in the real Core object